src/locale/kbd-model-map

```diff
@@ -1,5 +1,6 @@
 # Originally generated from system-config-keyboard's model list.
 # consolelayout		xlayout	xmodel		xvariant	xoptions
+arabic			ara,us	pc105		-		terminate:ctrl_alt_bksp,grp:shift_toggle
```

Comment:
Hmm, did this get added to `kbd`? I don't see anything with that name on my machine…

Reply:
It's provided by `kbd-legacy` here (openSUSE TW): `/usr/share/kbd/keymaps/i386/qwerty/arabic.map.gz`

Reply:
According to
https://build.opensuse.org/package/view_file/openSUSE:Factory/kbd/kbd.changes?expand=1
it seems to be a downstream thing.

Reply:
For `arabic` I'm currently seeing if openSUSE can switch to `ara` since the latter is the name used by the x11 layout already.

Reply:
what does SUSE want to use this for, btw? this is basically legacy RH stuff that got sucked into systemd out of system-config-keyboard (which was RH's old keyboard config tool). I'm not sure it's a good idea for anything that isn't using this stuff already to *start* using it. or are you trying to solve the 'know when to pick a non-converted console layout' problem?

This whole area is filled with sharks and sadness, and what we *really* need is for someone to finally come up with a way for consoles to use xkb layouts and xkb-style switching. no good ever comes of having to handle this awful xkb/kbd difference stuff.

Reply:
I guess it more or less is, yeah. We do it the other way around - the installer deals in xkb layouts, and relies on this ancient rune stuff to pick an appropriate console layout for whichever xkb layout(s) are selected.